### PR TITLE
Issue 165 : CreateOffer, DeleteOffer, getAllOFfers in Admin_VIP

### LINKED ISSUE #165: Admin_VIP
- [x] createOffer
- [x] editOffer
- [x] deleteOffer
- [x] getOffers (possible duplicate)
- [x] getOffer

### LINKED ISSUE #184: Pass the id value when booking the offer

### PR DESCRIPTION
**Pull request by eunbeek**
This is not completed work, please check the code and do not approve this PR.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (eunbeek) [inline .vscode/launch.json]: will delete this file in next pr

**Review by yuanLeeMidori — COMMENTED** [review]
@eunbeek 
getAll, create, delete all work fine, thank you for the good job.

However, when I click create, the page doesn't change or lead me to any other page. I didn't know if I created successfully or not, I still stay in the same create form page.

**Comment by eunbeek** [comment]
@yuanLeeMidori  

I fixed it, please re-check it one more. 
Edit one also is implemented now.

**Review by yuanLeeMidori — APPROVED** [review]
All good, thank you for the hard work.

**Review by rko7 — APPROVED** [review]

**Pull request merged**